tool/tsh/common/tsh.go

```diff
@@ -692,6 +695,7 @@ func Run(ctx context.Context, args []string, opts ...CliOption) error {
 		Default("true").
 		BoolVar(&cf.EnableEscapeSequences)
 	app.Flag("bind-addr", "Override host:port used when opening a browser for cluster logins").Envar(bindAddrEnvVar).StringVar(&cf.BindAddr)
+	app.Flag("callback", "Override the base URL (scheme://host:port) of the link shown when opening a browser for cluster logins. Must be used with --bind-addr.").StringVar(&cf.CallbackAddr)
```

Comment:
There is probably more user education needed than this.  Maybe we should require the user to confirm the action manually.  Some possible risks:
* Users need to understand the level of trust they are extending to this host
* A fake blog could default this callback as a means of stealing credentials if the code is copy / pasted
* Scripts including this could pose similar risks, requiring a manual approval helps mitigate this inclusion

Thoughts?